include/mini/common.h

```diff
@@ -6,11 +6,20 @@
 #include <string.h>
 #include <errno.h>
 
+#define BUFSIZE 4096 
+
+#ifdef _WIN32
+#include <Windows.h>
+#define ABORT ExitProcess(42)
+#else
+#define ABORT abort()
```

Comment:
We have but it makes allot of noise (Tries to find a debugger and creates a useless coredump)